Gum/Managers/NameVerifier.cs

```diff
@@ -293,25 +299,38 @@ public bool IsBehaviorNameValid(string behaviorName, BehaviorSave behaviorSave,
         }
         return string.IsNullOrEmpty(whyNotValid);
     }
-    private void IsNameValidCommon(string name, out string whyNotValid)
+    public static bool IsNameValidCommon(string name, out string whyNotValid)
```

Comment:
I think this can stay non-static, with the service injected

Reply:
I did some work to inject this object throughout a few plugins, but making this specific property static will take more work so I'm going to postpone that.